packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -110,8 +110,8 @@ test.describe('Call Composite E2E CallPage Tests', () => {
 
       await page.goto(buildUrl(serverUrl, user));
       await waitForCallCompositeToLoad(page);
-      // Move the mouse off-screen to avoid tooltips / delayed focus from introducing flake in the snapshots
-      await page.mouse.move(0, 0);
+      // we need to dismiss any tool tips if there are any so we press the keyboard.
+      await page.keyboard.down('Escape');
```

Comment:
Shouldn't this be https://playwright.dev/docs/api/class-keyboard#keyboard-press

Also, isn't this racy: The escape might be pressed before the tooltip shows, and then fail to dismiss the tooltip later?